local Library = loadstring(game:HttpGet("https://raw.githubusercontent.com/deividcomsono/Obsidian/refs/heads/main/Library.lua"))()
local Window = Library:CreateWindow({
    Title = "XM",
    Footer = "зүҲжң¬: V1 | жёёжҲҸ :йҖҡз”Ё",
    Icon = 106760189301805,
    NotifySide = "Right",
})
local Tab = Window:AddTab({
    Name = "йҰ–йЎө",
    Description = "BROBROBROBROBRO",
    Icon = "home"
})
local LeftGroupBox = Tab:AddLeftGroupbox("дё»йЎө", "wrench")
LeftGroupBox:AddInput("MyPseudoCodeInput", {
    Text = "й¬јз•ңи§Ҷйў‘ж’ӯж”ҫрҹ’Җрҹ’Җрҹ’Җ",
    Default = "https://github.com/CN-ZX114514/XM-Code/raw/refs/heads/main/%E7%9C%BC%20%E7%9D%9B%20%E7%9E%8E%20%E5%9C%A8%20%E6%89%8B%20%E6%9C%BA%20%E4%B8%8A_%E5%93%94%E5%93%A9%E5%93%94%E5%93%A9_bilibili.mp4",
    Placeholder = "hhttps://github.com/CN-ZX114514/XM-Code/raw/refs/heads/main/%E7%9C%BC%20%E7%9D%9B%20%E7%9E%8E%20%E5%9C%A8%20%E6%89%8B%20%E6%9C%BA%20%E4%B8%8A_%E5%93%94%E5%93%A9%E5%93%94%E5%93%A9_bilibili.mp4",
    Callback = function(value)
        print("Value changed", value)
    end
})
LeftGroupBox:AddVideo("MyVideo", {
    Video = "https://github.com/CN-ZX114514/XM-Code/raw/refs/heads/main/%E7%9C%BC%20%E7%9D%9B%20%E7%9E%8E%20%E5%9C%A8%20%E6%89%8B%20%E6%9C%BA%20%E4%B8%8A_%E5%93%94%E5%93%A9%E5%93%94%E5%93%A9_bilibili.mp4",
    Looped = true,
    Playing = true,
    Volume = 0,
})
LeftGroupBox:AddDivider("иҝҷжҳҜжҲ‘д»¬зҡ„DC(жңүжјҸжҙһеҸҜд»ҘеҸҚйҰҲ)")
LeftGroupBox:AddButton("еӨҚеҲ¶(жҮ’еҫ—еҶҷи„ҡжң¬зӣҙжҺҘеј№еҮә)", function()
    Library:Notify("https://discord.com/invite/9GFpDKXrXu", 4)
    loadstring("setclipboard('https://discord.com/invite/9GFpDKXrXu")()
end)
local RightGroupBox = Tab:AddRightGroupbox("ж”ҜжҢҒжёёжҲҸ","wrench")
RightGroupBox:AddLabel("[рҹҹў]йҖҡз”Ё")
RightGroupBox:AddLabel("[рҹҹў]еҝҚиҖ…дј еҘҮ")
local RightGroupBox = Tab:AddRightGroupbox("зј–еҶҷдәәе‘ҳдёҺжөӢиҜ•дәәе‘ҳ","wrench")
RightGroupBox:AddLabel("[<font color=\"rgb(73, 230, 133)\">жүҖжңүиҖ…</font>] NB123 ")
RightGroupBox:AddLabel("[<font color=\"rgb(73, 230, 133)\">жёёжҲҸжү§иЎҢеҷЁ</font>] е…ғе®қAI ")
RightGroupBox:AddLabel("[<font color=\"rgb(73, 230, 133)\">жөӢиҜ•дәәе‘ҳ</font>] NB123 ")
local Tab = Window:AddTab({
    Name = "еҠҹиғҪ",
    Description = "BROBROBROBROBRO",
    Icon = "wrench"
})
local LeftGroupBox = Tab:AddLeftGroupbox("еҠҹиғҪ", "wrench")
LeftGroupBox:AddSlider("MySlider", {
    Text = "йҖҹеәҰ",
    Default = 16,
    Min = 0,
    Max = 100,
    Rounding = 0,
    Callback = function(speed)
    Humanoid.WalkSpeed:speed
    end
})
LeftGroupBox:AddSlider("MySlider", {
    Text = "и·іи·ғ",
    Default = 16,
    Min = 0,
    Max = 100,
    Rounding = 0,
    Callback = function(speed)
    Humanoid.JumpHeight:speed
    end
})

local RightGroupBox = Tab:AddRightGroupbox("еҗҺй—Ё")
RightGroupBox:AddButton({
    Text = "LALOL-Hub-Backdoor",
    Func = function()
        loadstring(game:HttpGet("https://rawscripts.net/raw/Universal-Script-LALOL-Hub-Backdoor-158177"))()
    end
})
local Tab = Window:AddTab({
    Name = "и„ҡжң¬дёӯеҝғ",
    Description = "BROBROBROBROBRO",
    Icon = "wrench"
})
